core/src/test/java/com/sap/ai/sdk/core/client/ExecutionUnitTest.java

```diff
@@ -107,14 +131,163 @@ void testPostExecution() {
+                    """
+                    {
+                      "configurationId": "e0a9eb2e-9ea1-43bf-aff5-7660db166676"
+                    }
+                    """)));
+  }
+
+  @Test
+  void getExecutionById() {
+    wireMockServer.stubFor(
+        get(urlPathEqualTo("/lm/executions/e529e8bd58740bc9"))
+            .withHeader("AI-Resource-Group", equalTo("default"))
+            .willReturn(
+                aResponse()
+                    .withStatus(HttpStatus.SC_OK)
+                    .withHeader("content-type", "application/json")
+                    .withBody(
+                        """
+                        {
+                          "completionTime": "2024-09-09T19:10:58Z",
+                            "configurationId": "218b651d-113d-4da0-be15-b63a644a92fb",
+                            "configurationName": "i749902_exec_conf",
+                            "createdAt": "2024-09-09T19:09:57Z",
+                            "executableId": "wt-spam-detection-i749902",
+                            "id": "e529e8bd58740bc9",
+                            "modifiedAt": "2024-09-09T19:11:17Z",
+                            "outputArtifacts": [
+                              {
+                                "createdAt": "2024-09-09T19:10:48Z",
+                                "description": "",
+                                "executionId": "e529e8bd58740bc9",
+                                "id": "c4792df8-da67-44fe-ad99-b5ea74bbb248",
+                                "kind": "model",
+                                "modifiedAt": "2024-09-09T19:10:48Z",
+                                "name": "classifier-model-output",
+                                "scenarioId": "scenario-spam-detection-i749902",
+                                "url": "ai://default/e529e8bd58740bc9/classifier-model-output"
+                              }
+                            ],
+                            "scenarioId": "scenario-spam-detection-i749902",
+                            "startTime": "2024-09-09T19:10:11Z",
+                            "status": "COMPLETED",
+                            "submissionTime": "2024-09-09T19:10:11Z",
+                            "targetStatus": "COMPLETED"
+                        }
+                        """)));
+
+    final AiExecutionResponseWithDetails execution =
+        new ExecutionApi(getClient(destination)).executionGet("default", "e529e8bd58740bc9");
+
+    assertThat(execution).isNotNull();
+    assertThat(execution.getCompletionTime()).isEqualTo("2024-09-09T19:10:58Z");
+    assertThat(execution.getConfigurationId()).isEqualTo("218b651d-113d-4da0-be15-b63a644a92fb");
+    assertThat(execution.getConfigurationName()).isEqualTo("i749902_exec_conf");
+    assertThat(execution.getCreatedAt()).isEqualTo("2024-09-09T19:09:57Z");
+    // executableId is not in the generated client.
+    assertThat(execution.getCustomField("executableId")).isEqualTo("wt-spam-detection-i749902");
```

Comment:
You should report to AI Core that they have the field `executionScheduleId` instead of `executableId`